package.json

```diff
@@ -22,16 +22,16 @@
     "build-production-client": "NODE_ENV=production BABEL_ENV=production yarn build-client",
     "build-production": "yarn distclean && yarn && gulp languages:extract && yarn build-production-client",
     "build-languages": "gulp languages",
-    "docker:up": "docker-compose -f docker/docker-compose.yml up",
+    "docker:up": "docker-compose -f docker/docker-compose.yml up -d",
     "docker:stop": "docker-compose -f docker/docker-compose.yml stop",
     "docker:down": "docker-compose -f docker/docker-compose.yml down",
     "docker:build": "docker-compose -f docker/docker-compose.yml build",
-    "docker:wp": "docker container exec jetpack_wordpress wp --allow-root --path=/var/www/html/",
-    "docker:install": "yarn docker:wp core install --url=localhost --title='Hello World' --admin_user=wordpress --admin_password=wordpress --admin_email=wordpress@example.com --skip-email && yarn docker:wp plugin activate jetpack",
-    "docker:multisite-convert": "yarn docker:wp core multisite-convert && yarn docker:wp config set DOMAIN_CURRENT_SITE 'DOCKER_REQUEST_URL' --raw --type=constant && cp ./docker/htaccess-multi ./docker/wordpress/.htaccess",
-    "docker:sh": "docker exec -it jetpack_wordpress bash",
-    "docker:phpunit": "docker container exec jetpack_wordpress phpunit --configuration=/var/www/html/wp-content/plugins/jetpack/phpunit.xml.dist",
-    "docker:tail": "tail -f docker/logs/php/errors.log",
+    "docker:wp": "docker-compose -f docker/docker-compose.yml run --rm wordpress wp --allow-root --path=/var/www/html/",
+    "docker:install": "yarn -s docker:wp core install --url=localhost --title=Hello --admin_user=wordpress --admin_password=wordpress --admin_email=wordpress@example.com --skip-email && yarn docker:wp plugin activate jetpack",
+    "docker:multisite-convert": "yarn -s docker:wp core multisite-convert && yarn docker:wp config set DOMAIN_CURRENT_SITE 'DOCKER_REQUEST_URL' --raw --type=constant && cp ./docker/htaccess-multi ./docker/wordpress/.htaccess",
+    "docker:sh": "docker-compose -f docker/docker-compose.yml run --rm wordpress bash",
+    "docker:phpunit": "docker-compose -f docker/docker-compose.yml run --rm wordpress phpunit --configuration=/var/www/html/wp-content/plugins/jetpack/phpunit.xml.dist",
+    "docker:tail": "docker-compose -f docker/docker-compose.yml run --rm wordpress bash -c \"tail -f /var/www/html/wp-content/debug.log\"",
```

Comment:
I updated all of these to use docker-compose instead of raw docker commands. This means you can check out jetpack into a folder not named `jetpack` and have it still work. I also updated the tail command to use docker so that it works in places where `tail` may not be installed/available

Reply:
The thing about the folder naming does not apply here because we're having the container and image names fixed to `jetpack_wordpress` on the yml file. 

Apparently you replaced all of the `exec` commands to be `run`s and this is a different behaviour than the expected originally. With `run` we're starting new containers. The `--rm` flag does remove them but it seems like an overkill here.

Reply:
I updated those scripts to be `exec` again but keeping the `docker-compose` command in 94d300b90c4a0288d84e9e9822f74bc63317a2f4. Let's chat if you think having `run` is better here.

Reply:
A consequence of running a script like `docker:sh` with `run` is that every change you attempt on that shell that's not related to a mounted volume won't be available to the already running container  launched via `up`.